### PR TITLE
[DOCFIX] Update configuration documentation

### PR DESCRIPTION
**Pull request by ZacBlanco**

### CODE REVIEW

File: docs/en/operation/Configuration.md [hunk 1]
```diff
@@ -20,16 +21,19 @@ The two major components to configure are
 Customizing how an application interacts with Alluxio is specific to each application.
 The following are recommendations for some common applications.
 
-Note that it is only valid to set client-side configurations for applications,
-such as properties prefixed with `alluxio.user`.
-Similarly, setting server-side properties on a compute application has no effect,
-such as properties prefixed with either `alluxio.master` or `alluxio.worker`.
+Note that properties prefixes with `alluxio.user` are only valid to be set on machines which are
```
Comment (ns1123) [inline docs/en/operation/Configuration.md]: Is this correct with Cluster Defaults?

File: docs/en/operation/Configuration.md [hunk 2]
```diff
@@ -125,13 +129,12 @@ $ cp conf/alluxio-env.sh.template conf/alluxio-env.sh
...
-they will initialize their own Alluxio configuration properties with the default values
-supplied by the masters based on the master-side `${ALLUXIO_HOME}/conf/alluxio-site.properties` files.
-As a result, cluster admins can set client-side settings (e.g., `alluxio.user.*`),
-or network transport settings (e.g., `alluxio.security.authentication.type`),
-or worker settings (e.g., `alluxio.worker.*`)
-in `${ALLUXIO_HOME}/conf/alluxio-site.properties` on all the masters,
+or Alluxio workers connect to an Alluxio master, they will initialize their own Alluxio
+configuration properties with the default values supplied by the masters based on the master-side
+`${ALLUXIO_HOME}/conf/alluxio-site.properties` files.
+As a result, cluster admins can set client-side settings (e.g., `alluxio.user.*`), or network
```
Comment (ns1123) [inline docs/en/operation/Configuration.md]: "cluster admins can set default client-side settings"

File: docs/en/operation/Configuration.md [hunk 3]
```diff
@@ -125,13 +129,12 @@ $ cp conf/alluxio-env.sh.template conf/alluxio-env.sh
...
+transport settings (e.g., `alluxio.security.authentication.type`), or worker settings
```
Comment (ns1123) [inline docs/en/operation/Configuration.md]: Cluster defaults no longer propagates worker settings

File: docs/en/operation/Configuration.md (hunk 1 above)
Comment (ZacBlanco) [reply docs/en/operation/Configuration.md]: done

File: docs/en/operation/Configuration.md (hunk 2 above)
Comment (ZacBlanco) [reply docs/en/operation/Configuration.md]: updated

File: docs/en/operation/Configuration.md (hunk 3 above)
Comment (ZacBlanco) [reply docs/en/operation/Configuration.md]: updated

File: docs/en/operation/Configuration.md [hunk 4]
```diff
@@ -125,14 +127,13 @@ $ cp conf/alluxio-env.sh.template conf/alluxio-env.sh
...
-or worker settings (e.g., `alluxio.worker.*`)
-in `${ALLUXIO_HOME}/conf/alluxio-site.properties` on all the masters,
-which will be distributed and become cluster-wide default values when clients and workers connect.
+or Alluxio workers connect to an Alluxio master, they will initialize their own Alluxio
+configuration properties with the default values supplied by the masters based on the master-side
+`${ALLUXIO_HOME}/conf/alluxio-site.properties` files.
+As a result, cluster admins can set default client-side settings (e.g., `alluxio.user.*`), or
+network transport settings (e.g., `alluxio.security.authentication.type`) in
+`${ALLUXIO_HOME}/conf/alluxio-site.properties` on all the masters, which will be distributed and
+become cluster-wide default values when clients and workers connect.
```
Comment (ns1123) [inline docs/en/operation/Configuration.md]: remove workers from this line

File: docs/en/operation/Configuration.md [hunk 5]
```diff
@@ -142,32 +143,44 @@ master's `alluxio-site.properties` file.
...
 through Alluxio client, unless `${ALLUXIO_HOME}/conf` is on applications' classpath.
 
 ### Path Defaults
 
-Since version 2.0, Alluxio administrators can set default client side configurations for Alluxio paths.
-FileSystem client operations have options, FileSystem options are derived from client side configuration
-properties. Only these configuration properties can be set as as path defaults.
+Since version 2.0, Alluxio administrators can set default client side configurations for Alluxio
+paths.
+Filesystem client operations have options, Filesystem options are derived from client side
```
Comment (ns1123) [inline docs/en/operation/Configuration.md]: grammar? Make it two sentences, use a semi-colon, merge?

File: docs/en/operation/Configuration.md [hunk 6]
```diff
@@ -142,32 +143,44 @@ master's `alluxio-site.properties` file.
...
-If the administrator updates path defaults by
-`bin/alluxio fsadmin pathConf add --property alluxio.user.file.writetype.default=THROUGH /tmp`,
-afterwards, all `createFile` will by default have write type `THROUGH`.
+If the administrator updates path defaults using
+
+```console
+$ bin/alluxio fsadmin pathConf add --property alluxio.user.file.writetype.default=THROUGH /tmp
+```
+
+afterwards, all write operations will use the `THROUGH` write type by default.
```
Comment (ns1123) [inline docs/en/operation/Configuration.md]: "all write operations on that path"

File: docs/en/operation/Configuration.md (hunk 4 above)
Comment (ZacBlanco) [reply docs/en/operation/Configuration.md]: AFAIK alluxio workers contain an Alluxio client and will get cluster defaults for those from the master, so this is still correct.

ex:
https://github.com/Alluxio/alluxio/blob/master/core/server/worker/src/main/java/alluxio/worker/block/BlockMasterSync.java#L107-L108

File: docs/en/operation/Configuration.md (hunk 5 above)
Comment (ZacBlanco) [reply docs/en/operation/Configuration.md]: updated

File: docs/en/operation/Configuration.md (hunk 6 above)
Comment (ZacBlanco) [reply docs/en/operation/Configuration.md]: updated

**Comment by ZacBlanco** [comment]
This is a docfix, so I am not going to bother with CI checks on this PR

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please

**Comment by ZacBlanco** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**